man/man1/exiv2.1

```diff
@@ -681,7 +684,7 @@ the YYYY:MM:DD HH:MM:SS standard.
 .sp 1
 .ne 40
 .SH CONFIGURATION FILE
-Exiv2 can read an optional configuration file ~/.exiv2 on Unix systems and %USERPROFILE%\\exiv2.ini on Windows (using a Visual Studio build).  Cygwin and MinGW/msys2 follow the unix convention and use ~/.exiv2  You can determine the location of the configuration file with the command:
```

Comment:
the feature change to look in the current working directory was only merged into `main` so this line of  doc should just remain unchanged

Reply:
@hassec.  I've fixed this.  Thanks for pointing this out.  Good Catch.